deploy/helms/legion/templates/jenkins/jenkins-rbac.yaml

```diff
@@ -5,18 +5,15 @@ apiVersion: rbac.authorization.k8s.io/v1beta1
 metadata:
   name: jenkins
 rules:
-- apiGroups: [""]
-  resources: ["pods"]
-  verbs: ["create","delete","get","list","patch","update","watch"]
-- apiGroups: [""]
-  resources: ["pods/exec"]
-  verbs: ["create","delete","get","list","patch","update","watch"]
-- apiGroups: [""]
-  resources: ["pods/log"]
-  verbs: ["get","list","watch"]
-- apiGroups: [""]
-  resources: ["secrets"]
-  verbs: ["get"]
+- apiGroups: [""] # "" indicates the core API group
+  resources: ["pods", "deployments", "replicasets", "services", "pods/log", "pods/exec", "secrets"]
```

Comment:
Does Jenkins really need all these permissions?

Reply:
We can remove deployments, replicasets and services, but I'm not sure about secrets (Jenkins passes some secrets to slaves)

Reply:
Does it use K8S secrets for that?

Reply:
https://github.com/jenkinsci/kubernetes-plugin/blob/f6cff5d7e9ce9da3279660159e0cb064efac534f/src/main/kubernetes/service-account.yml